diff --git a/src/include/matahari/agent.h b/src/include/matahari/agent.h
index 8345841..1489ed2 100644
--- a/src/include/matahari/agent.h
+++ b/src/include/matahari/agent.h
@@ -26,8 +26,6 @@
 #define __MATAHARI_DAEMON_H
 
 #include <string>
-#include <qpid/sys/Time.h>
-#include <qpid/agent/ManagementAgent.h>
 #include <qpid/management/Manageable.h>
 #include <qpid/messaging/Connection.h>
 #include <qmf/AgentEvent.h>
diff --git a/src/lib/CMakeLists.txt b/src/lib/CMakeLists.txt
index 610040f..c3978ab 100644
--- a/src/lib/CMakeLists.txt
+++ b/src/lib/CMakeLists.txt
@@ -46,7 +46,7 @@ if(WITH-QMF)
     add_library (mcommon_qmf SHARED mh_agent.cpp)
     set_target_properties(mcommon_qmf PROPERTIES SOVERSION 1.0.0)
 
-    target_link_libraries(mcommon_qmf mcommon ${QPIDCOMMON_LIBRARY} ${QPIDCLIENT_LIBRARY} ${QPIDMESSAGING_LIBRARY} ${QMF2_LIBRARY})
+    target_link_libraries(mcommon_qmf mcommon ${QPIDMESSAGING_LIBRARY} ${QMF2_LIBRARY})
     if(QPIDTYPES_LIBRARY)
         target_link_libraries(mcommon_qmf ${QPIDTYPES_LIBRARY})
     endif(QPIDTYPES_LIBRARY)
diff --git a/src/lib/mh_agent.cpp b/src/lib/mh_agent.cpp
index 7edb217..43677ec 100644
--- a/src/lib/mh_agent.cpp
+++ b/src/lib/mh_agent.cpp
@@ -35,9 +35,6 @@ int use_stderr = 0;
 #include <stdlib.h>
 #include <cstdlib>
 
-#include <qpid/sys/Time.h>
-#include <qpid/agent/ManagementAgent.h>
-#include <qpid/client/ConnectionSettings.h>
 // #include <qpid/sys/ssl/util.h>
 #include <qmf/DataAddr.h>
 #include "matahari/agent.h"
@@ -53,8 +50,6 @@ extern "C" {
 #endif
 }
 
-using namespace qpid::management;
-using namespace qpid::client;
 using namespace std;
 
 typedef qpid::types::Variant::Map OptionsMap;
