diff --git a/src/lib/CMakeLists.txt b/src/lib/CMakeLists.txt
index b64e022..e33e659 100644
--- a/src/lib/CMakeLists.txt
+++ b/src/lib/CMakeLists.txt
@@ -57,7 +57,7 @@ if(WITH-QMF)
     set_target_properties(mcommon_qmf PROPERTIES SOVERSION 1.0.0)
 
     target_link_libraries(mcommon_qmf mcommon
-        ${QPIDCLIENT_LIBRARY} ${QPIDMESSAGING_LIBRARY} ${QMF2_LIBRARY}
+        ${QPIDMESSAGING_LIBRARY} ${QMF2_LIBRARY}
         ${nss_LIBRARIES})
     if(QPIDTYPES_LIBRARY)
         target_link_libraries(mcommon_qmf ${QPIDTYPES_LIBRARY})
